src/main/kotlin/org/jitsi/nlj/srtp/SrtpTransformer.kt

```diff
@@ -177,13 +178,34 @@ class SrtpDecryptTransformer(
     contextFactory: SrtpContextFactory,
     parentLogger: Logger
 ) : SrtpTransformer(contextFactory, parentLogger) {
+    var earlyDiscardedPacketsSinceLastSuccess = 0
+    private val disableBypass = maxConsecutivePacketsDiscardedEarly <= 0
```

Comment:
This is kinda a double-negative and makes reading the if below a bit hard.  What about:
`private val alwaysProcess = maxConsecutivePacketsDiscardedEarly > 0`?

Reply:
Thanks (and good job on not doing the error in my comment :)